lib/one_offs/tasks.rb

```diff
@@ -2,26 +2,28 @@
 
 namespace :one_offs do
   desc "Create a one off script"
-  task :create_one_off do
+  task :create do
     # Get command line arguments
     ARGV.each { |a| task a.to_sym do ; end }
 
     directory = Rails.root.join('lib', 'one_offs')
     filename = ARGV[1].singularize
 
     File.open("#{directory}/#{Time.now.to_i}_#{filename}.rb", "w") do |file|
-      contents = "class #{filename.classify.to_s}\n"\
-                 "  def self.process\n"\
-                 "    # Enter your migration code here....\n"\
+      contents = "class #{filename.classify}\n"\
```

Comment:
could we use a heredoc?

Reply:
tangential but I don't like the default spec it adds, which has the process in a before block. I'd rather test the change so I always end up deleting that bit. don't suppose you fancy changing that too?

Reply:
you can do heredoc with squiggles now like 
```
    <<~HEREDOC
      def this_is_still_indented
        correctly
    HEREDOC
```

Reply:
oh, good to know (for the future)